src/coreComponents/fileIO/silo/SiloFile.cpp

```diff
@@ -3167,5 +3167,21 @@ int SiloFile::getMeshType( string const & meshName ) const
   return meshType;
 }
 
+bool SiloFile::isFieldPlotEnabled( PlotLevel const wrapperPlotLevel,
```

Comment:
You could add a
```c++
bool SiloFile::isFieldPlotEnabled( string WrapperBase const & wrapper ) const
{
    return this->isFieldPlotEnabled( wrapper.getPlotLevel(), wrapper.getName() );
}
```
which can bring some smoother reading.

Reply:
Done, I implemented what you suggested